generator/src/templates/partials/apache.hbs

```diff
@@ -0,0 +1,59 @@
+<VirtualHost *:443>
+    ...
+    SSLEngine on
+{{#if (minver "2.4.8" form.serverVersion)}}
+    SSLCertificateFile      /path/to/signed_certificate_followed_by_intermediate_certificates
+{{else}}
+    SSLCertificateFile      /path/to/signed_certificate
+    SSLCertificateChainFile /path/to/intermediate_certificate
+{{/if}}
+    SSLCertificateKeyFile   /path/to/private/key
+
+    # Uncomment the following directive when using client certificate authentication
+    # SSLCACertificateFile    /path/to/ca_certs_for_client_authentication
+    {{#if form.hsts}}
+
+    # HTTP Strict Transport Security (mod_headers is required) ({{output.hstsMaxAge}} seconds)
+    Header always set Strict-Transport-Security "max-age={{output.hstsMaxAge}}"
+    {{/if}}
+    ...
+</VirtualHost>
+
+# {{form.config}} configuration, tweak to your needs
+SSLProtocol             {{#if (minver "2.3.16" form.serverVersion)}}{{replace output.protocols " -SSLv2" ""}}{{else}}{{output.protocols}}{{/if}}
+SSLCipherSuite          {{output.cipherSuites}}
+SSLHonorCipherOrder     on
+{{#if (minver "0.9.8" form.opensslVersion)}}
```

Comment:
Maybe through in some breadcrumbs to the docs that explain this complex conditional section
```
{{#if (minver "0.9.8" form.opensslVersion)}}
  // https://httpd.apache.org/docs/2.2/mod/mod_ssl.html#sslcompression
  {{#if (matchesminorver "2.2" form.serverVersion)}}
    {{#if (minver "2.2.24" form.serverVersion)}}
SSLCompression          off
    {{/if}}
  {{/if}}
  // https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslcompression
  {{#if (matchesminorver "2.4" form.serverVersion)}}
    {{#if (minver "2.4.3" form.serverVersion)}}
SSLCompression          off
    {{/if}}
  {{/if}}
{{/if}}
{{#if (minver "0.9.8f" form.opensslVersion)}}
  // https://httpd.apache.org/docs/2.2/mod/mod_ssl.html#sslsessiontickets
  {{#if (matchesminorver "2.2" form.serverVersion)}}
    {{#if (minver "2.2.30" form.serverVersion)}}
SSLSessionTickets       off
    {{/if}}
  {{/if}}
  // https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslsessiontickets
  {{#if (matchesminorver "2.4" form.serverVersion)}}
    {{#if (minver "2.4.11" form.serverVersion)}}
SSLSessionTickets       off
    {{/if}}
  {{/if}}
{{/if}}
```